WordPress/Resources/release_notes.txt

```diff
@@ -1,5 +1,7 @@
-We removed a bug in devices running iOS 16 or later, where typing in the block editor while dictating caused the dictated text to be lost. Phew.
+* [**] [internal] Blaze: Switch to using new canBlaze property to determine Blaze eligiblity. [#20916]
```

Comment:
My understanding is that Blaze is a Jetpack-only feature (see also internal ref peeHDf-zm-p2#comment-381). 

In doubt, I decided to leave this in the notes because it's marked as internal so it won't affect the user facing, editorialized release notes. 

cc @momo-ozawa as the author of the PR, just curious if you can confirm my understanding or point me to additional info. Thanks 🙇‍♂️

Reply:
@mokagio Blaze is indeed a Jetpack-only feature. I should have added the [Jetpack-only] tag - sorry!

Reply:
No worries. I think it's okay to omit when it comes to internal items anyways.